scripts/prohibit-properties-unused-keys

```diff
@@ -8,14 +8,21 @@ source "$self_dir"/utils.sh
 bundle=frontend/base/src/main/resources/GitMacheteBundle.properties
 [[ -f $bundle ]] || die "Bundle file ($bundle) has been (re)moved"
 
-exit_code=0
-for key in $(
-  grep -Po '.*(?==)' -- $bundle \
-  | grep -Evx 'action\.GitMachete\.[A-Za-z]+\.(description|GitMachete(ContextMenu|Toolbar).text|text)'
-); do
-  if ! git grep -qF "$key" -- '*.java' '*.kt' ":!$bundle"; then
-    echo "Property key $key defined in $bundle does not seem to be used anywhere"
-    exit_code=1
-  fi
-done
-exit $exit_code
+all_expected_keys=$(
+  grep --only-matching --perl-regexp '.*(?==)' -- $bundle \
```

Comment:
https://virtuslab.github.io/tips/#regex/lookbehind-lookahead, IMO the only advanced regex feature (assuming that capturing groups aren't considered advanced) you'll ever need